README.md

```diff
@@ -36,12 +41,16 @@ See the official [React Native website](https://facebook.github.io/react-native/
 ### Requirements
 
 In addition to the requirements from the [Getting Started guide](http://facebook.github.io/react-native/docs/getting-started.html):
-- [NPM 3.x](https://www.npmjs.com/package/npm)
 - [Visual Studio 2015 Community](https://msdn.microsoft.com/en-us/library/dd831853.aspx)
+  * ``choco install VisualStudio2015Community`` (this will take some time)
+  * ``choco install vs2015.3``
 - [Windows 10 SDK Build 10586](https://developer.microsoft.com/en-US/windows/downloads/windows-10-sdk)
+  * ``choco install windows-sdk-10.1``
 
 If this is your first time using UWP, you may be able to install the SDK by opening the [solution file](ReactWindows/ReactNative.sln) in the ReactWindows folder in Visual Studio. After opening the solution, right click the Solution in the Solution Explorer and select the option labeled "Install Missing Components". You may have to shutdown Visual Studio to continue the installation.
 
+If you receive a build error that the 'ChakraBridge' project's target architecture doesn't match, change the 'Any CPU' dropdown in the Visual Studio toolbar to 'x86' or 'x64' so a common target is selected that is supported by all the projects in the solution.
```

Comment:
Can you remove this?  I deleted all the Any CPU configurations in the .sln file.